docs/extras/use_cases/extraction/index.mdx

```diff
@@ -12,13 +14,106 @@ Examples of this include:
 - Extracting multiple rows to insert into a database from a long document
 - Extracting the correct API parameters from a user query
 
-This work is extremely related to [output parsing](/docs/modules/model_io/output_parsers/).
-Output parsers are responsible for instructing the LLM to respond in a specific format.
-In this case, the output parsers specify the format of the data you would like to extract from the document.
-Then, in addition to the output format instructions, the prompt should also contain the data you would like to extract information from.
+## Overview
+
+In LangChain we provide a few useful abstractions that help you leverage OpenAI [function calling](https://openai.com/blog/function-calling-and-other-api-updates) (`-0613` models) and which go a long way to avoid model hallucination. This is in general the best go-to to extract structured data from text using langchain. However there's other two options that you should also consider:
+
+1. This work is extremely related to [output parsing](docs/modules/model_io/output_parsers/). Output parsers are responsible for instructing the LLM to respond in a specific format. We suggest using output parsers when either you are not using OpenAI models or you want to have very close control over the parsing (e.g. checking that the output is a date in a specific datetime format).
+2. While the conventional extraction chain is good enough for basic structuring of response data,
+   when doing extraction you often want to extract more complicated or nested structures.
+   For a deep dive on extraction, we recommend checking out [`kor`](https://eyurtsev.github.io/kor/),
+   a library that uses the existing LangChain chain and OutputParser abstractions
+   but deep dives on allowing extraction of more complicated schemas.
+
+All this said, let's see how easy it is to quickly and accurately extract structured data with LangChain.
+
+## Example #1: using a JSON schema
```

Comment:
Maybe we re-title this to `Quickstart`: OAI function are probably the quickest way to get started.